nucypher/blockchain/economics.py

```diff
@@ -195,3 +195,6 @@ def deployment_parameters(self) -> Tuple[int, ...]:
         ]
 
         return tuple(map(int, deployment_parameters))
+
+
+Economics = Union[SlashingEconomics, TokenEconomics]
```

Comment:
Classy! although it may go away after the rebase...

Reply:
:heavy_check_mark: